cypress/component/rich-text/RichTextEditor.spec.ts

```diff
@@ -147,18 +147,8 @@ describe('Rich Text Editor', { viewportHeight: 2000, viewportWidth: 1000 }, () =
         .parent()
         .parent()
         .dragTo(() => richText.editor.findByText('some text.'));
-      if (Cypress.browser.name === 'firefox') {
-        richText.expectValue(doc(entryBlock(), paragraph, emptyParagraph()));
-      } else {
-        richText.expectValue(
-          doc(
-            block(BLOCKS.PARAGRAPH, {}, text('some')),
-            entryBlock(),
-            block(BLOCKS.PARAGRAPH, {}, text(' text.')),
-            emptyParagraph()
-          )
-        );
-      }
```

Comment:
When using chrome, why was it expected we drag an embed entry into the middle of a string? cc.: @z0al